joomla/joomla-cms
# [5.4] Bugfix for Issue #10140 and #40996 - Banner Modules using 'Select by keywords' set to 'Yes'

Pull Request for Issue #10140 and #40996 


### Summary of Changes
- Put keywords in a UserState variable com_content.articles.keywords
- Use UserState variable to get the keywords from the articles

### Testing Instructions taken from #10140 

-Create a banner, including some keywords (see publishing).
-Create a banner module an setup the option "Search by Meta Keyword" to "YES".
-Activate the module in any position let's say "banner" on Cassiopeia template
-Disable the cache for the banners module and save.

-Create a Content article and setup some keywords (see publishing) which will be used to display the banner accordingly.

- Test with Client set - seems to work only with client set

### Actual result BEFORE applying this Pull Request

No banners shown

### Expected result AFTER applying this Pull Request

Banners shown, acording to keywords

### Link to documentations
Please select:
